### PR TITLE
Rough Draft of Alasca Techtalk post

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+published: 2024-05-17
+description: ALASCA Tech Talk, Open Tech Meetup and BCIX Roundtable
+title: Events in the last Month
+keywords: [ Tech, Update, Event ]
+authors: [ Tassilo, Marcel ]
+image: 006_open_tech_1.webp
+---
+
+
+In den letzten Monaten ist viel passiert, das wichtigste unsere kern Infrastruktur steht.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:11]:
```suggestion
In den letzten Monaten ist viel passiert, das Wichtigste, unsere Kerninfrastruktur steht. 
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+authors: [ Tassilo, Marcel ]
+image: 006_open_tech_1.webp
+---
+
+
+In den letzten Monaten ist viel passiert, das wichtigste unsere kern Infrastruktur steht. 
+
+Der [ALASCA e.V.](https://alasca.cloud) hat uns deswegen eingeladen den [#15](https://alasca.cloud/en/alasca-tech-talk-15/) ALASCA Tech Talk zu halten. 
+In diesem Vortrag, erklären [Matthias Wählisch](https://netd.cs.tu-dresden.de/about/waehlisch) und [Tassilo Tanneberger](https://tanneberger.me) 
+BGP Grundlagen, welche Funktion ein Internet Exchagen erfüllt und wie der DD-IX konstruiert wurde.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:15]:
```suggestion
die Grundlagen von BGP, welche Funktion ein Internet Exchagen erfüllt und wie der DD-IX konstruiert wurde.
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+![Youtube Link](https://youtu.be/ieWTCBEq5IA)
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:17]:
```suggestion
Eine Aufzeichung findest du auf ![Youtube](https://youtu.be/ieWTCBEq5IA)!
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+----
+
+Einige Tage später haben wir das [erste DD-IX Open Tech Meetup](https://dd-ix.net/en/event/open-tech-meeting-2024-05) durchgeführt. Wir haben uns sehr gefreut einen Teil der Dresdner Netzwerkcommunity in den Räumlichkeiten der TU Dresden begrüsen zu können.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:21]:
```suggestion
Ein paar Tage später fand das [erste DD-IX Open Tech Meetup](https://dd-ix.net/en/event/open-tech-meeting-2024-05) statt. Wir haben uns sehr gefreut einen Teil der Dresdner Netzwerkcommunity in den Räumlichkeiten der TU Dresden begrüsen zu können.
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Wir bedanken uns hiermit ganz besonders bei [Thomas Liske](https://ibh.social/@liske) für die ausgezeichnete Präsentation über unsere Infrastruktur.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:22]:
```suggestion
Wir bedanken uns hiermit ganz besonders bei [Thomas Liske](https://ibh.social/@liske) für die ausgezeichnete Präsentation unsere Infrastruktur.
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+Dannach gab es viele Ideen und Überlegungen, welche es bei Getränken und Snacks zu diskutieren galt.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:23]:
```suggestion
Danach gab es bei Snacks und Getränken viele Anregungen und Diskusionen aller Beleiligten.
```

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+![](007_roundtable.webp)
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:25]: Add alt text (text in the brackets)

File: content/news/007.events-april-may-2024-05-17.de.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+Mitte Mai haben wir am BCIX Roundtable teilgenommen. Es war wieder ein Interessanter Abend mit Vorträgen und fröhlichen Austausch mit vielen bekannten und neuen Gesichern.
```
Comment (MarcelCoding) [inline content/news/007.events-april-may-2024-05-17.de.md:27]:
```suggestion
Zum Schluss haben wir mitte Mai am BCIX Roundtable teilgenommen. Es war wieder ein Interessanter Abend mit Vorträgen und fröhlichen Austausch mit vielen bekannten und neuen Gesichern.
```

File: content/blog/007.events-april-may-2024-05-17.de.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+description: ALASCA Tech Talk, Open Tech Meetup and BCIX Roundtable
+title: Events in the last Month
+keywords: [ Tech, Update, Event ]
+authors: [ Tassilo, Marcel ]
+image: 006_open_tech_1.webp
+---
+
+Der [ALASCA e.V.](https://alasca.cloud) hat uns deswegen eingeladen den [#15](https://alasca.cloud/en/alasca-tech-talk-15/) ALASCA Tech Talk zu halten. 
+In diesem Vortrag, erklären [Matthias Wählisch](https://netd.cs.tu-dresden.de/about/waehlisch) und [Tassilo Tanneberger](https://tanneberger.me) 
+die Grundlagen von BGP, welche Funktion ein Internet Exchagen erfüllt und wie der DD-IX konstruiert wurde.
```
Comment (Nimuedaertya) [inline content/blog/007.events-april-may-2024-05-17.de.md:12]: Internet Exchange Point

File: content/blog/007.events-april-may-2024-05-17.de.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+Eine Aufzeichung findest du auf ![Youtube](https://youtu.be/ieWTCBEq5IA)!
+
+
+Ein paar Tage später fand das [erste DD-IX Open Tech Meetup](https://dd-ix.net/en/event/open-tech-meeting-2024-05) statt. Wir haben uns sehr gefreut einen Teil der Dresdner Netzwerkcommunity in den Räumlichkeiten der TU Dresden begrüsen zu können.
+Wir bedanken uns hiermit ganz besonders bei [Thomas Liske](https://ibh.social/@liske) für die ausgezeichnete Präsentation unser Infrastruktur.
```
Comment (Nimuedaertya) [inline content/blog/007.events-april-may-2024-05-17.de.md:18]: unserer Infrastruktur

File: content/blog/007.events-april-may-2024-05-17.de.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+Dannach gab es viele Ideen und Überlegungen, welche es bei Getränken und Snacks zu diskutieren galt.
+
+
+![](007_roundtable.webp)
+
+
+Mitte Mai haben wir am BCIX Roundtable teilgenommen. Es war wieder ein Interessanter Abend mit Vorträgen und fröhlichen Austausch mit vielen bekannten und neuen Gesichern.
```
Comment (Nimuedaertya) [inline content/blog/007.events-april-may-2024-05-17.de.md:25]: interessanter
Gesichtern

File: content/blog/007.events-april-may-2024-05-17.en.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+description: ALASCA Tech Talk, Open Tech Meetup and BCIX Roundtable
+title: Events in the last Month
+keywords: [ Tech, Update, Event ]
+authors: [ Tassilo, Marcel ]
+image: 006_open_tech_1.webp
+---
+
+The [ALASCA e.V.](https://alasca.cloud) has therefore invited us to give the [#15](https://alasca.cloud/en/alasca-tech-talk-15/) ALASCA Tech Talk. 
+In this talk, [Matthias Wählisch](https://netd.cs.tu-dresden.de/about/waehlisch) and [Tassilo Tanneberger](https://tanneberger.me) explain 
+BGP basics, which function an Internet Exchagen fulfils and how the DD-IX was constructed.
```
Comment (Nimuedaertya) [inline content/blog/007.events-april-may-2024-05-17.en.md:12]: Internet Exchange points fulfills

**Review by Nimuedaertya — CHANGES_REQUESTED** [review]

**Pull request merged**